hs_core/models.py

```diff
@@ -2076,15 +2076,12 @@ def get_metadata_xml(self, pretty_print=True, include_format_elements=True):
 
     def _get_metadata(self, metatdata_obj):
```

Comment:
The method name and comment are not descriptive what's going on in this method.

Reply:
Can line 2058 be moved to the else statement in this method?

Reply:
This method has been removed.